### PR TITLE
Make view profile page normal 

### PR DESCRIPTION
**Pull request by AlexSukov**

### CODE REVIEW

File: app/assets/images/profile.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Magons) [inline app/assets/images/profile.svg]: svg? call me and we talk about this

File: app/views/users/show.html.haml [hunk 2]
```diff
@@ -1 +1,20 @@
...
+.row
+  .col-xs-12.col-sm-12.col-md-12.col-lg-12
+    %table.table.profile
+      %tr
+        %th{colspan: '2'}= @user.username
+      %tr
+        %td{colspan: '2'}= image_tag("profile.svg")
+      %tr
+        %td{colspan: '2'}= link_to 'Редактировать профиль', edit_user_registration_path, class: 'btn btn-link header-nav-link'
+      -if @user.comments!=[]
```
Comment (Magons) [inline app/views/users/show.html.haml]: `any?`

File: app/assets/images/profile.svg (hunk 1 above)
Comment (AlexSukov) [reply app/assets/images/profile.svg]: @Magons i deleted svg. But i''ll call you today. After work.

Comment (AlexSukov) [reply app/assets/images/profile.svg]: @Magons i deleted svg in branch 17, will delete it here

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
   end
 
   def show
     @user = User.find_by_username(params[:id])
   end
+
+  private
+
+  def set_comments
+    @comments = Comment.where(user_id: current_user.id)
```
Comment (Magons) [inline app/controllers/users_controller.rb]: current_user.comments

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
+    @posts = Post.where(id: 37)
```
Comment (Magons) [inline app/controllers/users_controller.rb]: current_user.posts

**Pull request merged**